BrickKit/bricks/src/main/java/com/wayfair/brickkit/brick/ViewModelBrick.java

```diff
@@ -0,0 +1,166 @@
+package com.wayfair.brickkit.brick;
+
+import android.databinding.DataBindingUtil;
+import android.databinding.ViewDataBinding;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.VisibleForTesting;
+import android.util.SparseArray;
+import android.view.View;
+
+import com.wayfair.brickkit.BrickViewHolder;
+import com.wayfair.brickkit.padding.BrickPadding;
+import com.wayfair.brickkit.size.BrickSize;
+
+/**
+ * This class is used as a Generic Brick that can take in any XML Layout and use DataBinding to
+ * insert information from a {@link ViewModel}.
+ */
+public class ViewModelBrick extends BaseBrick {
```

Comment:
I found it really helpful to have some other constructors like (ViewModel, int layoutId) for the cases where we just want to use the defaults

Reply:
I have made it follow the Builder Pattern, it only requires a layoutId, everything else is optional.